R/get.R

```diff
@@ -86,7 +89,12 @@ get_users <- function(src, page_size = 500, prefix = NULL, limit = Inf) {
 get_groups <- function(src, page_size = 500, prefix = NULL, limit = Inf) {
   validate_R6_class(src, "Connect")
 
-  res <- page_offset(src, src$groups(page_size = page_size, prefix = prefix), limit = limit)
+  if (!is.null(prefix)) {
```

Comment:
Since this is clearly working around a bug, can we make a followup issue to remove the workaround when the bug is fixed, and link it here? At a minimum, this deserves a comment as to why we're doing this nonstandard thing.

Reply:
I'll add a comment to the code explaining why we're doing the non-standard thing. I don't think it's actually a *bug* in Connect (see the response I'm about to write in the main comments thread on this issue in response to @tdstein)